src/FormPage/FormPage.tsx

```diff
@@ -119,7 +119,7 @@ const TextareaWordCountingWidget = (props) => {
   } = props;
 
   const showWordCount = !!schema.word_count;
-  const wordCount = value ? value.split(/\s+/g).length : 0;
+  const wordCount = value.split(/[\s]+/).filter(function (el) {return el !== '';}).length;
```

Comment:
Sometimes `value` is undefined (user hasn't filled out a field), so we still need the ternary check.

Also, for the `filter` function, we could rewrite it as `.filter(el => el !== '')` to reduce the line length.